diff --git a/Config/TasmanianConfig.in.cmake b/Config/TasmanianConfig.in.cmake
index 13fbd68..2a7ec5a 100644
--- a/Config/TasmanianConfig.in.cmake
+++ b/Config/TasmanianConfig.in.cmake
@@ -11,7 +11,7 @@ endif()
 # https://cmake.org/cmake/help/v3.5/module/CMakePackageConfigHelpers.html#module:CMakePackageConfigHelpers
 # seems to indicate that I need to pre-pend PACKAGE_ to the variable names after the @PACKAGE_INIT@ macro
 # but this doesn't seem to work, not sure if this is a "relocatable package" (low concern)
-include("@Tasmanian_final_install_path@/lib/@CMAKE_PROJECT_NAME@/@CMAKE_PROJECT_NAME@.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/Tasmanian.cmake")
@@ -26,7 +26,7 @@ if ("@Tasmanian_ENABLE_MPI@" AND NOT TARGET MPI::MPI_CXX)
 endif()
 
 add_executable(Tasmanian::tasgrid IMPORTED)
-set_property(TARGET Tasmanian::tasgrid PROPERTY IMPORTED_LOCATION "@Tasmanian_final_install_path@/bin/tasgrid${CMAKE_EXECUTABLE_SUFFIX_CXX}")
+set_property(TARGET Tasmanian::tasgrid PROPERTY IMPORTED_LOCATION "${PACKAGE_PREFIX_DIR}/tools/tasmanian/tasgrid${CMAKE_EXECUTABLE_SUFFIX}")
 
 add_library(Tasmanian::Tasmanian INTERFACE IMPORTED GLOBAL) # master target
 target_link_libraries(Tasmanian::Tasmanian INTERFACE Tasmanian_master)
